test/anmol.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+define(["d3", "src/form/Slider", "src/common/Icon", "src/other/Table", "src/other/Paginator"], function (d3, Slider, Icon, Table, Paginator) {
+    describe("widgets", function () {
+        this.timeout(10000);
+
+        describe("Slider Tests", function () {
+            var widget = { path: "src/form/Slider" };
```

Comment:
I am adding it to the class, so that all elements on the page have a class that points to their address. (Might be useful to write a selector later on). I can remove it, as I am not using it right now for anything.